install/tools/ipa-replica-conncheck

```diff
@@ -548,18 +603,17 @@ def main():
             time.sleep(3600)
             print_info("Connection check timeout: terminating listening program")
 
+
 if __name__ == "__main__":
     try:
         sys.exit(main())
-    except SystemExit as e:
-        sys.exit(e)
     except KeyboardInterrupt:
         print_info("\nCleaning up...")
         sys.exit(1)
-    except RuntimeError as e:
-        sys.exit(e)
```

Comment:
Sorry, my bad, except RuntimeError must stay there. The program is using RuntimeError to pass error messages to the user.